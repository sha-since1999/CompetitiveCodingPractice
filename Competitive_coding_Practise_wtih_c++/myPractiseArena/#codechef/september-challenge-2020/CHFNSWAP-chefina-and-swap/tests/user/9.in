3
094375023
65083745037
03947693