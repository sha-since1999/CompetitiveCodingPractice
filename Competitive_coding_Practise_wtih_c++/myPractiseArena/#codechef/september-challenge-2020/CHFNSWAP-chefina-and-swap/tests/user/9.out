27641805
0
0
